[2024-09-29T11:02:38.711+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-29T11:02:38.748+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: intialization_database.merge_transform_all_sources manual__2024-09-29T11:02:09.590498+00:00 [queued]>
[2024-09-29T11:02:38.760+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: intialization_database.merge_transform_all_sources manual__2024-09-29T11:02:09.590498+00:00 [queued]>
[2024-09-29T11:02:38.763+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-29T11:02:38.779+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): merge_transform_all_sources> on 2024-09-29 11:02:09.590498+00:00
[2024-09-29T11:02:38.787+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=124) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-29T11:02:38.789+0000] {standard_task_runner.py:63} INFO - Started process 126 to run task
[2024-09-29T11:02:38.790+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'intialization_database', 'merge_transform_all_sources', 'manual__2024-09-29T11:02:09.590498+00:00', '--job-id', '2780', '--raw', '--subdir', 'DAGS_FOLDER/initialization.py', '--cfg-path', '/tmp/tmp275lz3a1']
[2024-09-29T11:02:38.792+0000] {standard_task_runner.py:91} INFO - Job 2780: Subtask merge_transform_all_sources
[2024-09-29T11:02:38.850+0000] {task_command.py:426} INFO - Running <TaskInstance: intialization_database.merge_transform_all_sources manual__2024-09-29T11:02:09.590498+00:00 [running]> on host e78fe06b6548
[2024-09-29T11:02:38.965+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='intialization_database' AIRFLOW_CTX_TASK_ID='merge_transform_all_sources' AIRFLOW_CTX_EXECUTION_DATE='2024-09-29T11:02:09.590498+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-29T11:02:09.590498+00:00'
[2024-09-29T11:02:38.967+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-29T11:02:45.416+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-29T11:02:45.417+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-29T11:02:45.433+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=intialization_database, task_id=merge_transform_all_sources, run_id=manual__2024-09-29T11:02:09.590498+00:00, execution_date=20240929T110209, start_date=20240929T110238, end_date=20240929T110245
[2024-09-29T11:02:45.499+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-29T11:02:45.528+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T11:02:45.531+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
